src/protobufs/src/user-service/user-service.proto

```
@@ -10,15 +10,31 @@ service UserService {
   rpc RefreshToken(RefreshTokenRequest) returns (Session);
   rpc RequestAuthentication(RequestAuthenticationRequest) returns (Empty);
   rpc AuthenticateWithOTP(OTPAuthenticationRequest) returns (Session);
+  rpc GetActiveSessions(GetActiveSessionsRequest)
+      returns (GetActiveSessionsResponse);
+  rpc InvalidateSession(InvalidateSessionRequest) returns (Empty);
```

Buf (1.61.0)

"service.user.Empty" is used as the request or response type for multiple RPCs.

(RPC_REQUEST_RESPONSE_UNIQUE)